apps/react-18-tests-v9/jest-react-18.config.js

```diff
@@ -0,0 +1,83 @@
+// @ts-check
+const fs = require('node:fs');
+const path = require('node:path');
+
+const { pathsToModuleNameMapper } = require('ts-jest');
+const tsConfigBase = require('../../tsconfig.base.json');
+const tsPathAliases = pathsToModuleNameMapper(tsConfigBase.compilerOptions.paths, {
+  prefix: `<rootDir>/../../`,
+});
+
+const jestConfig = require('./jest.config');
+
+/**
+ * @type {import('@jest/types').Config.InitialOptions}
+ */
+const config = {
+  ...jestConfig,
+  moduleNameMapper: { ...tsPathAliases },
```

Comment:
in order to make path aliases work this needs to be overriden as root is different